R/mod_tableGen_utils.R

```diff
@@ -563,14 +563,57 @@ tg_guide <- cicerone::Cicerone$
   step(
     "download_table",
     "Keep a copy for your records",
-    "Download a table in CSV or HTML format as seen in the app. RTF support coming soon!"
+    "Download a table in RTF, CSV, or HTML format as seen in the app."
   )$
   step(
     "tableGen_ui_1-tblcode",
     "Produce this table outside of the app",
     "Have more data coming? Don't download the table, download the R script needed to produce the table. Then there's no need to even open up the app periodically to re-create your outputs!"
   )
 
+#' Create a `gt` table object
+#' 
+#' Create a `gt` table object from the data that will then be used to generate
+#' output in specific formats
+#' 
+#' @param data The data frame used to create the table
+#' @param input_table_title The Shiny input with the table title
+#' @param input_table_footnote The Shiny input with the table footnote(s)
+#' @param col_names A vector of column names
+#' @param col_total A vector of column totals
+#' @param subtitle The table subtitle
+#'  
+#' @export
```

Comment:
No need to `@export` right now - please remove and re-run `run_dev.R`. There is another function used in the reproducible script called `tg_gt()` that builds the gt object for the R script. We can potentially look at editing that in a future PR so that the user can create an RTF from an R script.

Reply:
@AARON-CLARK  Changed `create_gt_table()` to not exported. Ready for re-review.